ocamlformat-help.txt

```diff
@@ -146,6 +146,11 @@ OPTIONS (CODE FORMATTING STYLE)
            lines between one-liners of similar sorts. The default value is
            sparse.
 
+       --no-break-infix-before-func
+           Breaks infix expressions before the operator if it does not fit on
+           a single line and the operator is followed by an anonymous
+           function.
```

Comment:
The documentation should be reworded to reflect the default value. Perhaps: "Break infix operators whose right arguments are anonymous functions specially: do not break after the operator so that the first line of the function appears docked at the end of line after the operator"